Sources/Foundation/URLQueryItem.swift

```diff
@@ -7,96 +7,15 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
-
-/// A single name-value pair, for use with `URLComponents`.
-public struct URLQueryItem: ReferenceConvertible, Hashable, Equatable {
-    public typealias ReferenceType = NSURLQueryItem
-
-    fileprivate var _queryItem: NSURLQueryItem
-
-    public init(name: String, value: String?) {
-        _queryItem = NSURLQueryItem(name: name, value: value)
-    }
-
-    fileprivate init(reference: NSURLQueryItem) { _queryItem = reference.copy() as! NSURLQueryItem }
-    fileprivate var reference: NSURLQueryItem { return _queryItem }
-
-    public var name: String {
-        get { return _queryItem.name }
-        set { _queryItem = NSURLQueryItem(name: newValue, value: value) }
-    }
-
-    public var value: String? {
-        get { return _queryItem.value }
-        set { _queryItem = NSURLQueryItem(name: name, value: newValue) }
-    }
-
-    public func hash(into hasher: inout Hasher) {
-        hasher.combine(_queryItem)
-    }
-
-    public static func ==(lhs: URLQueryItem, rhs: URLQueryItem) -> Bool {
-        return lhs._queryItem.isEqual(rhs._queryItem)
-    }
-}
-
-extension URLQueryItem: CustomStringConvertible, CustomDebugStringConvertible, CustomReflectable {
-    public var description: String {
-        if let v = value {
-            return "\(name)=\(v)"
-        } else {
-            return name
-        }
-    }
-
-    public var debugDescription: String {
-        return self.description
-    }
-
-    public var customMirror: Mirror {
-        var c: [(label: String?, value: Any)] = []
-        c.append((label: "name", value: name))
-        c.append((label: "value", value: value as Any))
-        return Mirror(self, children: c, displayStyle: .struct)
-    }
-}
+// URLQueryItem is defined in FoundationEssentials
+@_exported import FoundationEssentials
 
 extension URLQueryItem: _NSBridgeable {
     typealias NSType = NSURLQueryItem
-    internal var _nsObject: NSType { return _queryItem }
-}
-
-extension URLQueryItem: _ObjectiveCBridgeable {
```

Comment:
Looks like this change drops the `_ObjectiveCBridgeable` conformance here, was that intentional? I don't think we want to remove it, we'll likely just have to adjust the implementation to call the right initializers (we can use `_nsObject` and `_swiftObject` for convenience)

Reply:
`_ObjectiveCBridgeable` is already implemented in `FoundationEssentials`